### ISSUE TITLE
Remove search function

### ISSUE THREAD

**Issue by smoyth** [opendatakit/website#83]
It currently searches old blog content and returns the same large image for each result. This is not a site that needs a search function in my opinion. All the juicy content is on the docs, forum, and Github.

**Comment by lognaturel** [comment]
One possibility for it that I believe @downeym had suggested would be to do cross-site search to allow searching all the juicy content you mention at once. But I agree that in the mean time, removing the local site search seems better than having it.

**Comment by wbrunette** [comment]
I agree with @lognaturel. If we cannot get a multi-site search to work we should remove it. However, seems like a task to try to complete before the launch. Renamed the issue from "remove search function" to "either enhance search functionality or remove it".

**Comment by yanokwa** [comment]
I'll send in a PR to remove it and leave this issue as enhancing the search because I don't think that's going to be easy to do before launch.

**Comment by smoyth** [comment]
I agree it sounds like a hard problem to solve well.

**Comment by wbrunette** [comment]
I think I have a solution, I am working on it this afternoon :)

**Comment by wbrunette** [comment]
I am holding off for more discussion, but to me automation is helpful for most people. Search has been shown to be incredibly useful. I think people being able to search and find answers is fine without people having to direct them. Ideally directing them with awesome documentation is the best but this is an even harder challenge. Giving them a tool to narrow their search to ODK docs, forum, website seems like an easy automated helper. Like people searching on the internet people can be discerning and learn from their search.

**Comment by lognaturel** [comment]
> Why do we have search on the forum then? Shouldn't the same reasoning apply?

Agreed that it's starting to be less useful, unfortunately. That's something we can look at separately -- maybe there's some kind of filtering possible.

**Comment by yanokwa** [comment]
This doesn't feel like a priority for launch because no one will be confused by a search being added later. 

I'd rather we put our energy towards other issues. https://github.com/opendatakit/docs/issues/436 comes to mind as a really critical issue that no one is owning currently.

**Comment by wbrunette** [comment]
I think search will be an opportunity for people to self-help themselves and by creating a search of our current websites it will help users find the answers they are looking for. Thereby improving things.

**Comment by adammichaelwood** [comment]
> I'd rather we put our energy towards other issues. opendatakit/docs#436 comes to mind as a really critical issue that no one is owning currently.

That's on me. I'll update the issue.
